packages/jbrowse-web/src/jbrowseModel.js

```diff
@@ -88,17 +88,23 @@ export default function JBrowseWeb(
       },
       addAssemblyConf(assemblyConf) {
         const { name } = assemblyConf
-        if (!name) throw new Error('Can\'t add assembly with no "name"')
-        if (self.assemblyNames.includes(name))
-          throw new Error(
-            `Can't add assembly with name "${name}", an assembly with that name already exists`,
-          )
+        if (!name) {
+          throw new Error('Can\'t add assembly with no "name"')
+        }
+        const assembly = self.assemblies.find(asm => asm.name === name)
```

Comment:
This tries to avoid adding duplicate assemblies which results in downstream errors